index.bs

```diff
@@ -1219,7 +1219,7 @@ following Web IDL.
     dictionary CollectedClientData {
         required DOMString           challenge;
         required DOMString           origin;
-        required DOMString           hashAlg;
+        required DOMString           hashAlgorithm;
```

Comment:
Since this doesn't go on the wire, I'd prefer the spelled-out form. Not willing to die on the hill though.